app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html

```diff
@@ -87,7 +88,7 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_import.arriving.h3")</h
     </ul>
 
 
-    @if(isInbound) {
+    @if(direction.equals(UK_INBOUND)) {
```

Comment:
Can we use `==` (or `===`) here instead? It's more idiomatic Scala and handles nulls better (not that null is a problem in this case)

Reply:
updated.